<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>temperature</class>
 <widget class="QDialog" name="temperature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pretvornik v Celzije</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="inTemp"/>
   </item>
   <item>
    <widget class="QLabel" name="lbRez">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
